c-example/README.md

```diff
@@ -0,0 +1,55 @@
+# BridgeStan from C
+
+This shows how one could write a C program which calls BridgeStan.
+
+Any compiled language with a C foreign function interface and
+the ability to link against C libraries should be able to work similarly.
+
+## Usage - dynamic linking
```

Comment:
Usage - dynamic -> Usage with dynamic

and same for static case below.  A colon could also work.